components.go

```diff
@@ -195,7 +195,7 @@ func (attendee *Attendee) ParticipationStatus() ParticipationStatus {
 
 func (attendee *Attendee) getPropertyFirst(parameter Parameter) string {
 	vs := attendee.getProperty(parameter)
-	if vs != nil && len(vs) > 0 {
+	if len(vs) > 0 {
```

Comment:
I feel that since this is ensuring that it's /not/ null it's different. Basically it returns an empty string, rather than returning a NPE. I will have to push back on this one.

Reply:
Same as above. But the `nil` check is also not needed because `len(vs)` will be `0`
 if `vs == 0`.
 https://play.golang.org/p/5bjyfO5OmB

Reply:
Ah.. Sorry when I read it I just assumed the `>` was a `==`. You know what they say about assumptions.. Only the best make them. :P Or something like that right?